lib/galaxy/webapps/galaxy/controllers/history.py

```diff
@@ -543,8 +546,7 @@ def display_structured(self, trans, id=None):
             'template': html_template
         }
 
-    @web.expose
-    @web.json
+    @expose_api
```

Comment:
I guess this should probably be `expose_api_anonymous`, since the old code explicitly handled the case that there is no trans.user